docker/px4-dev/Dockerfile_ros-kinetic

```diff
@@ -39,6 +40,8 @@ RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-key 421C365BD9F
 	&& apt-get clean autoclean \
 	# pip
 	&& pip install --upgrade numpy px4tools pymavlink \
+	&& python3 -m pip install --user --upgrade pip setuptools wheel \
```

Comment:
The docker build is done as root, which is why I didn't bother with `--user` for the other pip installs. I think this is generally frowned upon for pip usage, but in this case it's probably better to have it installed system wide and not worry about PATH updates.

Reply:
More of a question or note than critical feedback:
Is it necessary to upgrade pip? Also, why the switch to `python3 -m pip` from `pip3`? I understand that this would look through `PYTHON_PATH`, which could be important as you upgrade pip, but I'd think that we should be fine with the package we just installed.

Reply:
@dagar you're right, I removed the `--user` argument and upgrading pip was also not necessary, thank you @lamping7

Reply:
generally, I noticed that building the image takes a very long time, probably due to installing all the ros stuff. @dagar could we maybe use an image that has a lot of the ros stuff pre-installed instead of an ubuntu base image? I'm using an image with a specific python version pre-installed internally avoiding all these issues. it's not necessary though, regarding that re-building probably doesn't happen often.